src/snowflake/snowpark/_internal/analyzer/complexity_stat.py

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) 2012-2024 Snowflake Computing Inc. All rights reserved.
+#
+
+from enum import Enum
+
+
+class ComplexityStat(Enum):
+    FILTER = "filter"
+    ORDER_BY = "order_by"
+    JOIN = "join"
+    SET_OPERATION = "set_operation"  # UNION, EXCEPT, INTERSECT, UNION ALL
+    SAMPLE = "sample"
+    PIVOT = "pivot"
+    UNPIVOT = "unpivot"
+    WINDOW = "window"
+    GROUP_BY = "group_by"
+    PARTITION_BY = "partition_by"
+    CASE_WHEN = "case_when"
+    LITERAL = "literal"
+    COLUMN = "column"
+    FUNCTION = "function"
+    IN = "in"
+    LOW_IMPACT = "low_impact"
```

Comment:
so you want to have both "LOW_IMPACT" and "OTHERS"?

Reply:
i think we can have both, then we have a category for each node in the plan tree node, it would be easy for us to calculate the total number of nodes in the tree